docs/getting-started/rbac.rst

```diff
@@ -0,0 +1,478 @@
+=======================================================
+Role-based Access Control for Application Authorization
+=======================================================
+
+When managing access to resources within an application, it can be useful to
+group permissions into **roles**, and assign these roles to users. This is
+known as **Role-based Access Control (RBAC).**
+
+While there is no one way to implement RBAC, there are two types of
+relationships that must be considered in a roles system:
+
+1. **User-role relationships** define what roles a user has. The relationship
+could be a direct assignment from user-to-role, but could also be an indirect
+relationship that depends on additional information.
+2. **Role-permission relationships** define the access permissions that a
+role grants to a user. In oso, permissions generally consist of an **action**
+and a **resource.**
+
+oso represents these relationships as rules in a policy file, written in our
+declarative logic programming language called **Polar**. In general, a policy
+specifies user-role relationships with ``user_in_role`` rules and
+role-permission relationships with ``role_allow`` rules. The general form of
+an **allow rule** that calls these rules looks like this:
+
+.. code-block:: polar
+    :class: no-select
+
+    allow(actor, action, resource) if
+        actor_in_role(actor, resource, role) and
```

Comment:
shoud use `user_in_role` or `actor_in_role` for both

Reply:
Ah shit yeah I forgot I made that change. I think `actor_in_role` is more oso-y since we talk about actors everywhere else. But this whole doc is about users so idk.